7_drivers_munge/src/GCM_driver_utils.R

```diff
@@ -245,8 +236,13 @@ download_gcm_data <- function(out_file_template, query_geom,
   query_simplegeom <- sf_pts_to_simplegeom(query_geom_WGS84)
 
   # Build query_url
-  query_url <- sprintf("https://cida.usgs.gov/thredds/dodsC/notaro_%s_%s",
-                       gcm_name, gcm_projection_period)
+  query_url <- sprintf(
+    "http://gdp-netcdfdev.cr.usgs.gov:8080/thredds/dodsC/notaro_debias_%s",
+    tolower(gcm_name))
+
+  # Stop now if you aren't on VPN
+  if(!RCurl::url.exists(paste0(query_url, ".html")))
```

Comment:
I like this, but I think it is the only use of `RCurl` in this repo/pipeline. Seems like a useful thing to include and I don't see the same type of function in another library we're already using (e.g., `httr`) so :+1: to have on.